ci/generate-workflows.py

```diff
@@ -833,10 +833,10 @@ def tag(self):
     Target(name="centos9", container="quay.io/centos/centos:stream9"),
     Target(name="macos", os="macos-11"),
     # https://fedoraproject.org/wiki/End_of_life?rd=LifeCycle/EOL
-    Target(container="fedora:33"),
```

Comment:
Please also `git rm` the `gen_fedora33*` workflow files and add that to this PR!